packages/kit/src/core/sync/write_types.js

```diff
@@ -42,24 +48,36 @@ export function write_types(config, manifest_data) {
 				const content = fs.readFileSync(route.page.module, 'utf8');
 				const proxy = tweak_types(content, module_names);
 
-				if (proxy) {
-					// TODO only do this if the module does in fact export `load`
-					write(`${outdir}/proxy${path.basename(route.page.module)}`, proxy);
-					imports.push(`import { load } from './proxy+page.js';`);
+				if (proxy && proxy.exports.includes('load')) {
+					const basename = path.basename(route.page.module);
+					write(`${outdir}/proxy${basename}`, proxy.code);
+					imports.push(`import { load } from './proxy${basename}';`);
 					exports.push(`export type Data = Awaited<ReturnType<typeof load>>;`);
 				} else {
-					// TODO bail out write file with `any` types
+					exports.push(`export type Data = any;`);
```

Comment:
should this be unknown or never instead?

Reply:
yeah, should probably be `unknown`, but only if the process failed. Going to make it `null` if the process succeeded but there's no `load`